sphinx/ext/autodoc/importer.py

```diff
@@ -156,6 +156,17 @@ def import_object(modname, objpath, objtype='', attrgetter=safe_getattr, warning
 
     try:
         module = import_module(modname, warningiserror=warningiserror)
+        import os
```

Comment:
I think adding new code to here is a bit ugly. How about moving this to helper function?